src/main/java/bob/Triplet.java

```diff
@@ -0,0 +1,29 @@
+package bob;
+
+public class Triplet<T, U, V> {
```

Comment:
Cool to see you using generics here and making a triplet class. However, it could be more useful to name this class more purposefully, "Task"?